libapicore/httpServer.cpp

```diff
@@ -18,19 +18,34 @@ using namespace eth;
 
 httpServer http_server;
 
-void httpServer::tableHeader(stringstream& ss, unsigned columns)
+void httpServer::tableHeader(stringstream& ss)
 {
     char hostName[HOST_NAME_MAX + 1];
     gethostname(hostName, HOST_NAME_MAX + 1);
     string l = m_farm->farmLaunchedFormatted();
-    ss << "<html><head><title>" << hostName
-       << "</title><style>tr:nth-child(even){background-color:Gainsboro;}</style>"
-          "<meta http-equiv=refresh content=30></head><body><table width=\"50%\" border=1 "
-          "cellpadding=2 cellspacing=0 align=center>"
-          "<tr valign=top align=center style=background-color:Gold><th colspan="
-       << columns << ">" << ethminer_get_buildinfo()->project_name_with_version << " on "
-       << hostName << " - " << l << "<br/>Pool: " << m_manager->getActiveConnectionCopy().Host()
-       << "</th></tr>";
+    ss << "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01//EN\" "
```

Comment:
I'd definitely change this to the HTML5 `doctype`. No need for us to us such an ancient doctype.

Reply:
It doesn't pass HTML5 spec.

Reply:
Well, clearly since old code is used in this file :P. But it's not that hard. If someone provides me with the HTML files we output so that I can test this, I can provide a patch to switch to HTML5.

Reply:
Stand by.